internal/setup/setup.go

```diff
@@ -150,18 +172,31 @@ func (r *Release) validate() error {
 			for newPath, newInfo := range new.Contents {
 				if old, ok := paths[newPath]; ok {
 					oldInfo := old.Contents[newPath]
-					if !newInfo.SameContent(&oldInfo) || (newInfo.Kind == CopyPath || newInfo.Kind == GlobPath) && new.Package != old.Package {
+					if !newInfo.SameContent(&oldInfo) || (newInfo.Kind == CopyPath || newInfo.Kind == GlobPath || newInfo.Kind == GeneratePath) && new.Package != old.Package {
 						if old.Package > new.Package || old.Package == new.Package && old.Name > new.Name {
 							old, new = new, old
 						}
 						return fmt.Errorf("slices %s and %s conflict on %s", old, new, newPath)
 					}
 				} else {
-					if newInfo.Kind == GlobPath {
+					if newInfo.Kind == GlobPath || newInfo.Kind == GeneratePath {
 						globs[newPath] = new
 					}
 					paths[newPath] = new
 				}
+				if newInfo.Kind == GeneratePath {
+					if old, ok := generate[newInfo.Generate]; ok {
```

Comment:
Update: we should not be validating that chisel-releases only has one `generate: manifest`. We should be validating that there are no clashes with the path and that, **at runtime** the selection of slices does not contain two `generate: manifest`.

Reply:
Done in https://github.com/letFunny/chisel/pull/11/commits/b7fa761f8e6e8db9a94b566f272f85ac6c245344.